content/en/docs/tasks/administer-cluster/migrate-api/_index.md

```diff
@@ -0,0 +1,217 @@
+
+* Migrate manifests and API clients to use the **networking.k8s.io/v1** API version, available since v1.19.
+* All existing persisted objects are accessible via the new API
+* Notable changes:
+
+#### IngressClass
+
+IngressClass in the **networking.k8s.io/v1beta1** API version will no longer be served
+
+* Migrate manifests and API clients to use the **networking.k8s.io/v1** API version, available since v1.19.
+* All existing persisted objects are accessible via the new API
+
+#### RBAC
+
+ClusterRole, ClusterRoleBinding, Role, RoleBinding in the **rbac.authorization.k8s.io/v1beta1** API version will no longer be served
+
+* Migrate manifests and API clients to use the **networking.k8s.io/v1** API version, available since v1..
+* All existing persisted objects are accessible via the new API
+* Notable changes:
+
+#### PriorityClass
+
+PriorityClass in the **scheduling.k8s.io/v1beta1** API version will no longer be served
+
+* Migrate manifests and API clients to use the **scheduling.k8s.io/v1** API version, available since v1..
+* All existing persisted objects are accessible via the new API
+* Notable changes:
+
+#### Storage
+
+CSIDriver, CSINode, StorageClass, VolumeAttachment in the **storage.k8s.io/v1beta1** API version will no longer be served
+
+* Migrate manifests and API clients to use the **storage.k8s.io/v1** API version, available since v1.19.
+* All existing persisted objects are accessible via the new API
+* Notable changes:
+
+### v1.25
+
+Event in the **events.k8s.io/v1beta1** API version will no longer be served
+
+* Migrate manifests and API clients to use the **events.k8s.io/v1** API version, available since v1..
+* All existing persisted objects are accessible via the new API
+* Notable changes:
+
+RuntimeClass in the **node.k8s.io/v1beta1** API version will no longer be served
+
+* Migrate manifests and API clients to use the **node.k8s.io/v1** API version, available since v1..
+* All existing persisted objects are accessible via the new API
+* Notable changes:
+
+## What To Do
+
+Kubernetes 1.22 will be released later in 2021, so be sure to audit
+your configuration and integrations now!
+
+### Test with deprecated APIs disabled
+
+You can test your clusters by starting an apiserver with specific API versions disabled,
+to simulate the upcoming removal. Add the following flag to the apiserver startup arguments:
+
+`--runtime-config=$group/$version=false`
+
+For example:
+
+`--runtime-config=admissionregistration.k8s.io/v1beta1=false,apiextensions.k8s.io/v1beta1,...`
+
+### Locate use of deprecated APIs
+
+Use [client warnings, metrics, and audit information available in 1.19+]https://kubernetes.io/blog/2020/09/03/warnings/#deprecation-warnings
+to locate use of deprecated APIs.
+
+### Migrate to GA APIs
+
+* Update custom integrations and controllers to call the GA APIs
+* Change YAML files to reference the GA APIs
+
+    You can use the `kubectl-convert` command to automatically convert an existing object:
```

Comment:
cc @soltysh - where is this tool available?

Reply:
Fixed in https://github.com/kubernetes/kubernetes/pull/99155